roles/cloud_amazon/tasks/main.yml

```diff
@@ -24,7 +24,6 @@
     name: Kubernetes
     description: Kubernetes YUM repo
     baseurl: https://packages.cloud.google.com/yum/repos/kubernetes-el7-x86_64
-    reposdir: /dev/shm/
```

Comment:
@avgils This could be an issue, any thoughts?

Reply:
Ask ICT if this is fixed, this was a shortcut because they could not fix this in time... Then we can use a generic way to add the repo